RIOT-OS/RIOT
# makefiles/docker.inc.mk: pass PARTICLE_MONOFIRMWARE to docker env vars

### Contribution description

This PR allows to pass `PARTICLE_MONOFIRMWARE` to `DOCKER_ENV_VARS`
This will allow the generation of the bin-checksummed file needed by `dfu-util` flashtool and thus allow to flash the all particle-based board with RUN_IN_DOCKER=1

### Testing procedure


You can test it without owning the hardware.
`make BUILD_IN_DOCKER=1 BOARD=particle-xenon PARTICLE_MONOFIRMWARE=1 -C examples/hello-world clean `
`make BUILD_IN_DOCKER=1 BOARD=particle-xenon PARTICLE_MONOFIRMWARE=1 -C examples/hello-world flash `

On master, the .bin-checksummed isn't generated (check the output of `examples/hello-world/bin/particle-xenon`) Since this file is required to flash through dfu-util, flashing part fails.
With this PR, the file is generated and dfu-util try to flash the board

### Issues/PRs references
Fixes #16749
